Tools/src/BTagging.cc

```diff
@@ -6,8 +6,8 @@ namespace jmt {
   namespace BTagging {
     float discriminator_min(int wp, bool old) {
       assert(wp >= 0 && wp <= 2);
-      const float old_mins[3] = {0.5803, 0.8838, 0.9693}; // JMTBAD drop
-      const float mins[2][3] = { {0.0521, 0.3033, 0.7489}, {0.0494, 0.2770, 0.7264} };
+      const float old_mins[3] = {0.1355, 0.4506, 0.7738}; // JMTBAD drop
+      const float mins[2][3] = { {0.0532, 0.3040, 0.7476}, {0.0490, 0.2783, 0.7100} };
```

Comment:
This reminds me that we eventually need to add 2016 (everywhere) too! But that's a separate thing to worry about.